tool/tsh/db.go

```diff
@@ -49,11 +50,29 @@ func onListDatabases(cf *CLIConf) error {
 	if err != nil {
 		return trace.Wrap(err)
 	}
+
+	proxy, err := tc.ConnectToProxy(cf.Context)
+	if err != nil {
+		return trace.Wrap(err)
+	}
```

Comment:
I'm fixing a bug in Connect where we prematurely close the proxy client when fetching db users. I was looking at how tsh does things (since it was working fine in `tsh db ls`) and I landed here and noticed that we don't seem to close the proxy client here at all.

Shouldn't we have `defer proxy.Close()` at the end here just as we have `defer cluster.Close()` below?

Reply:
The fix in Connect I was working on: https://github.com/gravitational/teleport/pull/14230.